### PR TITLE
Feature/v1.11

### PR DESCRIPTION
**Pull request by dbmeneses**

### CODE REVIEW

File: sonar-scm-git-plugin/pom.xml [hunk 1]
```diff
@@ -18,9 +18,8 @@
   </scm>
  
   <properties>
-    <!-- Minimal supported version of SonarQube. It is defined in travis.sh 
-      for integration testing too. -->
-    <sonar.apiVersion>7.7-alpha1</sonar.apiVersion>
+    <!-- Minimal supported version of SonarQube -->
+    <sonar.apiVersion>7.9</sonar.apiVersion>
```
Comment (henryju) [inline sonar-scm-git-plugin/pom.xml]: Shouldn't you change `sonarQubeMinVersion` to 7.9 then?

File: sonar-scm-git-plugin/src/main/java/org/sonarsource/scm/git/JGitBlameCommand.java [hunk 2]
```diff
@@ -75,12 +74,8 @@ public void blame(BlameInput input, BlameOutput output) {
 
   private boolean cloneIsInvalid(File gitBaseDir) {
     if (Files.isRegularFile(gitBaseDir.toPath().resolve(".git/objects/info/alternates"))) {
-      LOG.warn("This repository references another local repository which is not supported. "
+      LOG.warn("This repository references another local repository which is not well supported. SCM information might be missing for some files. "
```
Comment (henryju) [inline sonar-scm-git-plugin/src/main/java/org/sonarsource/scm/git/JGitBlameCommand.java]: Isn't warn a bit too strong? Maybe info is enough?

File: sonar-scm-git-plugin/pom.xml (hunk 1 above)
Comment (dbmeneses) [reply sonar-scm-git-plugin/pom.xml]: good catch

File: sonar-scm-git-plugin/src/main/java/org/sonarsource/scm/git/JGitBlameCommand.java (hunk 2 above)
Comment (dbmeneses) [reply sonar-scm-git-plugin/src/main/java/org/sonarsource/scm/git/JGitBlameCommand.java]: I will change it to info

**Pull request merged**